### PR TITLE
The docs were using thenPresent but they should be using thenProceed

### PR DESCRIPTION
**Pull request by nickkaczmarek**
## Checklist:
- [x] Did you sign the [Contributor License Agreement](https://cla-assistant.io/wwt/SwiftCurrent)?
- [x] Did you [update the documentation](https://github.com/wwt/SwiftCurrent/blob/main/.github/CONTRIBUTING.md#documentation)?

### CODE REVIEW

**Review by morganzellers — APPROVED** [review]

**Comment by nickkaczmarek** [comment]
@morganzellers I was talking to @Tyler-Keith-Thompson about this before I merged and he mentioned that the api is actually `thenProceed(with:)` so I updated the examples.

**Comment by morganzellers** [comment]
@nickkaczmarek ah yes. My bad on that miss - I knew waiting to merge was a good choice.

File: .github/UPGRADE_PATH.md [hunk 1]
```diff
@@ -101,21 +101,21 @@ Our directions are written for only 1 major version upgrade at a time, as we hav
...
   }
   ```
-  
+
   ### The way you Test has Changed
   You used to be able to re-assign `proceedInWorkflow` to assert it was called with the args you expected, this has now slightly changed.
-  To get the *exact* behavior as before use `_proceedInWorkflow` to re-assign that closure. 
+  To get the *exact* behavior as before use `_proceedInWorkflow` to re-assign that closure.
   There's also `proceedInWorkflowStorage` which gives you the `AnyWorkflow.PassedArgs` used when `proceedInWorkflow` was called.
-
```
Comment (morganzellers) [inline .github/UPGRADE_PATH.md:109]: I use a Markdown linter in VSCode and it haaaates when these spaces exist 😂, glad to see this lol

**Review by morganzellers — APPROVED** [review]

File: .github/UPGRADE_PATH.md (hunk 1 above)
Comment (nickkaczmarek) [reply .github/UPGRADE_PATH.md:109]: Yeah I could probably remove all of them, but my editor just removed the ones in the files I edited.

**Pull request merged**